### DISCUSSION TITLE
Status bar focus after opening file?

### DISCUSSION THREAD

**Discussion by hi5** [tannerhelland/PhotoDemon#468] [General]
I open photodemon via my preferred file manager and it is launched as "path to photodemon.exe" "path to file" 

What I notice is that the focus (sometimes) after opening the first file is on the px/in/cm/etc control in the status bar near the image dimensions rather than on the image which is what I would expect. The problem I have with this is that I start keyboard navigation (page up / down etc) and nothing happens because I'm sending those key strokes to the active control not Photodemon as such. 

Is there a setting that can help with this? I never want that control to have focus after opening a file, I just want to get editing, not up/downing the control list for px/in/etc

Image (blue border indicates it has focus):
![screenshot (2)](https://user-images.githubusercontent.com/1973899/208990255-e296a3a6-3ad3-4449-b148-8388f1c770db.png)

**Comment by tannerhelland** [comment]
Hello @hi5 !  Thank you for reporting this issue.

I have been unable to replicate this focus issue locally (using Windows Explorer), so it may be something specific to the file manager you are using.  Can you let me know which file manager you are using?

Either way, I will try to create a fix that ensures correct keyboard focus regardless of how an image is loaded.  I will report back when an updated nightly build is ready with the changes.

**Comment by hi5** [comment]
Cheers - I can also replicate it is using a batch file:
`c:\Portable\PhotoDemon\PhotoDemon.exe "c:\downloads\screenshot (2).png"`

Obviously I have the status bar visible in PD :) - v9.2 pre-alpha build 84 at the moment

File manager I'm using is Total Commander and my own AutoHotkey script to open files in various programs but it essence it does the same as the batch file above.

**Reply by tannerhelland** [reply]
Thank you for the quick reply and info on the batch file!

**Comment by tannerhelland** [comment]
I've just pushed a new nightly build that *hopefully* fixes this.  I haven't been able to test every supported Windows version (or with software other than Windows Explorer) but this does solve the issue for me when using a batch file.  

Whenever convenient, please let me know if the latest nightly also solves the problem when using Total Commander + AutoHotkey.  Thank you!

**Comment by hi5** [comment]
Works for me, opened a dozen or so images from Total Commander + AutoHotkey and tried various batch files - all working correctly now.

Thank you very much and Happy Holidays! ☃

**Reply by tannerhelland** [reply]
Thank you so much for the fast reply.  Happy Holidays to you and yours as well 🎄
